src/gui/uithememanager.h

```diff
@@ -31,6 +31,7 @@
 
 #include <QColor>
 #include <QHash>
+#include <QIcon>
```

Comment:
you should check .cpp file as well, I mean you should remove this header from uithememanager.cpp file.